src/main/java/net/minecraftforge/event/LootTableLoadEvent.java

```diff
@@ -38,11 +39,13 @@
 {
     private final ResourceLocation name;
     private LootTable table;
+    private LootTableManager lootTableManager;
 
-    public LootTableLoadEvent(ResourceLocation name, LootTable table)
+    public LootTableLoadEvent(ResourceLocation name, LootTable table, LootTableManager lootTableManager)
```

Comment:
This is a breaking change, please keep the old constructor and deprecate it instead of removing it.